index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+<section class="Workssection1">
+    <header class="Header">
+        <a class="mylogo">MY logo</a>
+         <nav>
+            <a href="#"> </a>
+            <img href src="./image/logobuger.JPG">
+         </nav>
+    </header>
```

Comment:
- [ ] Impressive work @DamenaM. The one thing To improve here is the margins (top, bottom, left, right) defined in the design. The minimum mobile screen size your design should work well is `375`px. And right now, it's overflowing a bit right now.

### Your design on a `375`px screen. 
![Capture](https://user-images.githubusercontent.com/69077061/205219703-569a3d00-acff-4f8f-bf91-2a5455762632.PNG)